### PR TITLE
setup: upgrade invenio dependencies

### PR DESCRIPTION
**Pull request by audrium**
closes https://github.com/reanahub/reana-server/issues/479

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -10,6 +10,7 @@ Version 0.9.0 (UNRELEASED)
 - Adds REANA specification validation utilities.
 - Changes API rate limiter error messages to be more verbose.
 - Changes workflow scheduler to allow defining the checks needed to assess whether the cluster can start new workflows.
+- Changes to the latest versions of Invenio dependencies.
```
Comment (mdonadoni) [inline CHANGES.rst:13]: What do you think about `Updates the Invenio dependencies to the latest versions`?

**Review by mdonadoni — APPROVED** [review]
LGTM! 🎉

**Comment by mdonadoni** [comment]
There is also a little typo in the commit description: `#477` should be `#479`

File: CHANGES.rst (hunk 1 above)
Comment (audrium) [reply CHANGES.rst:13]: I agree that your suggestion sounds better, but there is a convention for the changelog to only use `changes/adds/fixes` as a prefix. @tiborsimko, I guess we should document it in REANA wiki?

**Pull request merged**